python/cuxfilter/dashboard.py

```diff
@@ -96,6 +97,19 @@ def _create_app(
     return server
 
 
+def _check_if_duplicates(charts):
+    _charts = [i.name for i in charts]
+    dups = list((Counter(_charts) - Counter(set(_charts))).keys())
+    if len(dups) > 0:
+        print(
```

Comment:
`warning.warn`? 

Also I think `Counter` can be applied moe simply:
```

In [11]: from collections import Counter

In [12]: data = ["foo", "bar", "baz", "foo"]

In [13]: dups = [k for k, v in Counter(data).items() if v > 1]

In [14]: dups
Out[14]: ['foo']

Reply:
Also `if dups` will suffice

Reply:
thanks, fixed this. Also using warnings.warn now